include/swift/Frontend/FrontendOptions.h

```diff
@@ -549,8 +541,18 @@ class FrontendOptions {
     return InputKind == InputFileKind::IFK_Swift &&
            Inputs.hasUniqueInputFilename();
   }
+private:
+  const char *computeSuffix();
```

Comment:
Suffix for what? Why non-`const`?

Reply:
Renamed to suffixForPrincipleOutputFileAccordingToAction & made it static.

Reply:
Nitpick: "principal"!

Reply:
D'Oh! Thanks for catching this. Fixed.